### ISSUE TITLE
`list`: add `--table` / `-t` flag to print a table with headers and metadata

### ISSUE THREAD

**Issue by embik** [embik/kubeconfig-bikeshed#12]
Depends on #6.

It would be nice to have a `kbs list --table` mode that prints information about the kubeconfigs in a formatted way. This could be piped into `fzf` in the future so that fuzzy searching is also applied to labels (or other metadata).
